docker-compose.yml

```diff
@@ -28,6 +28,9 @@ services:
       - type: bind
         source: $HOME/.ssh/known_hosts
         target: /root/.ssh/known_hosts
+      - type: bind
+        source: $HOME/logs/dashi/dashi.log
```

Comment:
Please update the README to create the `$HOME/logs/dashi/dashi.log` directories & file before running Docker-Compose

Reply:
Done, also used the env variable